api/db-cdr/generate-cdr/libproject/genomic_manifests.rb

```diff
@@ -385,37 +436,66 @@ def build_manifests_for_aw4_section(input_section, ingest_bucket, dest_bucket, d
     output_manifest = []
   end
 
+  prev_manifest = _read_previous_manifest(
+    project,
+    dest_bucket,
+    input_section["deltaReleaseManifestPath"],
+    input_section["deltaRelease"],
+    input_section["pooledDestPathInfix"])
+
   copy_manifest = aw4_rows.flat_map do |aw4_entry|
     unless output_manifest.nil?
-      out_row = {"person_id" => aw4_entry["research_id"]}
-      input_section["outputManifestSpec"].each do |aw4_col, out_col|
-        row = _build_copy_manifest_row(aw4_entry[aw4_col], ingest_base_path, destination, input_section, aw4_entry["research_id"])
-        destination_uri = File.join(row[:destination_dir], File.basename(row[:ingest_path]))
-        out_row[out_col] = destination_uri
+      out_row = { "person_id" => aw4_entry["research_id"] }
+      # If the research_id already exists in the prev manifest, just put as is in the new output manifest
+      unless prev_manifest.nil? or prev_manifest[aw4_entry["research_id"]].nil?
+        output_manifest.push(prev_manifest[aw4_entry["research_id"]].to_h)
+      else
+        input_section["outputManifestSpec"].each do |aw4_col, out_col|
+          row = _build_copy_manifest_row(aw4_entry[aw4_col], ingest_base_path, destination, input_section, aw4_entry["research_id"])
+          destination_uri = File.join(row[:destination_dir], File.basename(row[:ingest_path]))
+          out_row[out_col] = destination_uri
+        end
+        output_manifest.push(out_row)
       end
-      output_manifest.push(out_row)
     end
 
     source_paths = input_section["aw4Columns"].map { |k| aw4_entry[k] }
     source_paths.map do |source_path|
-      _build_copy_manifest_row(source_path, ingest_base_path, destination, input_section, aw4_entry["research_id"])
+      if prev_manifest.nil? or prev_manifest[aw4_entry["research_id"]].nil?
```

Comment:
Build the copy row only if the previous manifest doesn't have this research id. Otherwise the file doesn't need to be copied over because it will be used from the old location